pkg/cmd/docgen/diagrams.go

```diff
@@ -572,6 +572,17 @@ var specs = []stmtSpec{
 		inline: []string{"opt_drop_behavior", "table_name_list"},
 		match:  []*regexp.Regexp{regexp.MustCompile("'DROP' 'VIEW'")},
 	},
+	{
+		name:    "experimental_audit",
+		stmt:    "alter_onetable_stmt",
+		inline:  []string{"audit_mode", "alter_table_cmd", "alter_table_cmds"},
+		match:   []*regexp.Regexp{regexp.MustCompile("'ALTER' 'TABLE' relation_expr 'EXPERIMENTAL_AUDIT' 'SET' 'READ' 'WRITE'")},
```

Comment:
This could be reduced to just `EXPERIMENTAL_AUDIT` I think. But also fine as is.

Reply:
Thanks - IIRC it was needed to reduce the output to only the line starting with that particular `ALTER TABLE` combo, since the default output was quite large -- it required that and the  `replace` map to get it whittled down to what was needed for the diagram.  If there is a better way to get there I would be happy to learn it for future reference tho, this was the output of a guess-and-check process. :-)